forte/processors/__init__.py

```diff
@@ -18,3 +18,4 @@
 from forte.processors.text_generation_processor import *
 from forte.processors.vocabulary_processor import *
 from forte.processors.bert_based_query_creator import *
+from forte.processors.spacy_processors import *
```

Comment:
Similarly, we currently did not find a way to put 3rd party processors in __init__, since they might not get installed.